charts/pro-ui/values.yaml

```
@@ -9,60 +9,72 @@ dgctlDockerRegistry: ''
-  # @param ui.nodeSelector Kubernetes [node selectors](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector).
-  # @param ui.affinity Kubernetes pod [affinity settings](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity).
-  # @param ui.tolerations Kubernetes [tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) settings.
-  # @param ui.podAnnotations Kubernetes [pod annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/).
-  # @param ui.podLabels Kubernetes [pod labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/).
-  # @param ui.annotations Kubernetes [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/).
-  # @param ui.labels Kubernetes [labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/).
-  # @param ui.readinessProbe.enabled Enable [readinessProbe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes) on PRO UI containers
-  # @param ui.livenessProbe.enabled Enable [livenessProbe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes) on PRO UI containers
-  # @param ui.healthcheckPath Application http path for health check
-  # @param ui.containerPort Port on which application listen connection in container
-
-  replicas: 1
-  revisionHistoryLimit: 3
-  terminationGracePeriodSeconds: 60
-  nodeSelector: {}
-  affinity: {}
-  tolerations: []
-  podAnnotations: {}
-  podLabels: {}
-  annotations: {}
-  labels: {}
-  readinessProbe:
-    enabled: true
-  livenessProbe:
-    enabled: false
-  healthcheckPath: /api/healthcheck/app
-  containerPort: 3000
-
-  # @section Deployment settings
-
-  # @param ui.image.repository Repository
-  # @param ui.image.tag Tag
-  image:
-    repository: 2gis-on-premise/pro-ui
-    tag: 2.15.1
-
-  # @section UI service settings
+# @section Strategy settings
+
+# @param strategy.type Type of Kubernetes deployment. Can be `Recreate` or `RollingUpdate`.
+# @param strategy.rollingUpdate.maxUnavailable Maximum number of pods that can be created over the desired number of pods when doing [rolling update](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#rolling-update-deployment).
+# @param strategy.rollingUpdate.maxSurge Maximum number of pods that can be unavailable during the [rolling update](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#rolling-update-deployment) process.
+
+strategy:
+  type: RollingUpdate
+  rollingUpdate:
+    maxUnavailable: 0
+    maxSurge: 1
+
+# @section Deployment settings
+
+# @param image.repository Repository
+# @param image.tag Tag
+image:
+  repository: 2gis-on-premise/pro-ui
+  tag: 2.15.1
```

Версия именджа меняется или нет?

```suggestion
  tag: 2.27.0
```

---

обновил